.github/workflows/ci.yml

```diff
@@ -36,14 +36,17 @@ jobs:
 
       - name: Install Poetry
         run: |
-          pip install wheel
-          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+          pip install --user poetry
+
+      - name: Setup macOS PATH
+        if: runner.os == 'macOS'
+        run: |
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
```

Comment:
Why this is needed separately? I think you can simply change the below line 
https://github.com/ManimCommunity/manim/blob/be6c35f8a62671a07fc0d4226c3108f52f8bc04f/.github/workflows/ci.yml#L107
to match this change.

Reply:
Nope, the problem is that the very next job, `Setup cache variables` requires `poetry` to be available.

Reply:
> Nope, the problem is that the very next job, Setup cache variables requires poetry to be available.

I see but why not move that block of code here or just add to the path temporarily as before?

Reply:
I thought about doing that, but I figured we had some reason for putting the dependency installation after setting up the caches.